frontend/src/i18n/en.json

```diff
@@ -11,6 +11,13 @@
     "remove": "Remove",
     "search": "Search"
   },
+  "forgottenpassword": {
```

Comment:
itt is lehet camelCase meg mindenhol ahol így van hivathozva a változóra... kivéve a path ott nem szokás így szerintem